src/styles/lists.styl

```diff
@@ -175,6 +175,11 @@ heavyShadow = 0 4px 24px 0 rgba(50, 54, 63, .08)
         &[aria-busy='true']:hover
             transform none
 
+.app-list-wrapper, .services-list-wrapper
+    width 100%
+    max-width 50rem
+    margin auto
```

Comment:
we should use utility classes instead of creating new stylus classes. 

You can use: 
https://docs.cozy.io/cozy-ui/styleguide/section-utilities.html#kssref-utilities-dimensions

Reply:
Ok, but I still need .app-list-wrapper for max-width 50rem